### PR TITLE
Add `uninstall` command to phpenv

### PR DESCRIPTION
**Pull request by henriquemoody**

### CODE REVIEW

**Comment by rogeriopradoj** [comment]
:+1:

File: bin/rbenv-uninstall [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    echo
+    echo "Installed versions:"
+    phpenv versions | grep -v system | sed 's,\*, ,g; s, (.*,,g'
+    echo
+  } 1>&2
+  exit 2
+fi
+
+if [[ ${CONFIRM} -eq 0 ]]; then
+  echo -n "Are you sure you want to remove version ‘${VERSION}’? [y/N]: "
```
Comment (GrahamCampbell) [inline bin/rbenv-uninstall]: bit strange having an upper case N and lower case y isn;t it?

Comment (henriquemoody) [inline bin/rbenv-uninstall]: Actually that's how many other commands works, it means that is N rather than y, if it was Y/n it was the opposite.

Comment (GrahamCampbell) [inline bin/rbenv-uninstall]: right, ok :)

**Comment by GrahamCampbell** [comment]
Ping @php-build/collaborators for review.

**Comment by henriquemoody** [comment]
@GrahamCampbell, looks like I'm the only one in @php-build/collaborators :stuck_out_tongue_closed_eyes:

**Comment by GrahamCampbell** [comment]
Ping @php-build/owners. :)

**Comment by henriquemoody** [comment]
@rogeriopradoj already said yes to that more than 72 ago, I think you already can merge this pull request.

**Comment by GrahamCampbell** [comment]
I'd rather have reviews from more people.

**Comment by GrahamCampbell** [comment]
Meh, merging.

**Comment by GrahamCampbell** [comment]
Reviews can still take place after merging, lol.

**Pull request merged**